parser.py

```diff
@@ -141,9 +147,12 @@ def parse(filename):
         i['spouseID'] = spouse
         i['children'] = children
         date = i.get('birt-date')
+        deathdate = i.get('deat-date')
+        if deathdate != None:
+            now = deathdate
```

Comment:
This changes the value of `now` for every single future person, you can't just reassign this value